modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc

```diff
@@ -0,0 +1,255 @@
+      stableService: argo-rollouts-stable-service # must match name of Service above #<2>
+      trafficRouting:
+        plugins:
+          argoproj-labs/openshift:          
+            routes: #<3>
+              - rollouts-demo-route  # must match name of Route, above
+      steps: # In this example, route 30%/60%/100% of traffic to canary
+      - setWeight: 30
+      - pause: {}
+      - setWeight: 60
+      - pause: {}
+  selector: # Contents of this field must match the same value in the canary and stable Services (same contents for both)
+    matchLabels:
+      app: rollouts-demo
+----
+<1> Rollout references the canary service
+<2> Rollout references the stable service
+<3> Rollout references the route
+.. Click *Create*. 
+.. In the *RolloutManager* tab, under the *RolloutManagers* section, verify that the *Status* field of the *RolloutManager* instance shows as *Phase: Available*.
++
+When the first instance of the Rollout resource is created, the Rollout regulates the amount of traffic to be directed towards the various versions of the application, i.e. the stable and the canary versions. In the initial instance, the creation of the Rollout resource routes 100% of the traffic towards the stable version of the application and skips the part where the traffic is sent to the canary version.
+
+.. To verify that the Route is directing 100% of the traffic towards the stable version of the application, navigate to *Networking* → *Routes* and view the *rollouts-demo-route* Route. Select the *YAML* tab to view the following snippet:
++
+.Example: `Rollout`
+[source,yaml]
+----
+kind: Route
+metadata:
+  name: rollouts-demo-route
+spec:
+  alternateBackends:
+  - kind: Service
+    name: argo-rollouts-canary-service
+    weight: 0 # 0% of traffic is directly to the canary
+  # (...)
+  to:
+    kind: Service
+    name: argo-rollouts-stable-service
+    weight: 100 # 100% of traffic is directed to stable.
+----
++
+You can view that 100% of the traffic is directed towards the stable version of the application and none is directed towards the canary version.
+
+. Simulate the new canary version of the application by modifying the container image deployed in the Rollout. In the Administrator perspective of the web console, navigate to *Operators* -> *Installed Operators* -> *Red Hat OpenShift GitOps* -> *Rollout*.
+
+. Select the existing *Rollout*, and modify the `.spec.template.spec.containers.image` value from `argoproj/rollouts-demo:blue` to `argoproj/rollouts-demo:yellow`. As a result, the container image deployed in the Rollout is modified and the Rollout initiates a new canary deployment.
+
+. As per the `setWeight` property defined in the `.spec.strategy.canary.steps` field of the Rollout, initially 30% of traffic to the Route reaches the canary, and 70% of traffic is directed towards the stable version of the application. After 30% of traffic is being directed to the canary version of the application, the Rollout will pause.
```

Comment:
```suggestion
+
[NOTE]
====
As per the `setWeight` property defined in the `.spec.strategy.canary.steps` field of the `Rollout` resource, initially 30% of traffic to the route reaches the canary version, and 70% of traffic is directed towards the stable version. the rollout is paused after 30% of traffic is directed to the canary version.
====
```
This is not a step, so maybe an admonition?
Also, this seems like it should be included when the user is creating the `Rollout` resource.

Reply:
Fixed as an admonition.